### PR TITLE
Review

### PR DESCRIPTION
**Pull request by anpolimus**
Code for Sprint 7 with 7 summits game.

### CODE REVIEW

File: docroot/themes/custom/twelve_carnation/src/scss/modules/twelve_app/seven-summits/views/mountains/_mountains-fonts.scss [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      line-height: 27px;
+      font-family: $cachet-book;
+      font-size: 18px;
+      font-weight: 400;
+      fill: white;
+
+      //@include media-breakpoint-up(md) {
+      //  line-height: 36px;
+      //  font-size: 24px;
+      //}
```
Comment (podarok) [inline docroot/themes/custom/twelve_carnation/src/scss/modules/twelve_app/seven-summits/views/mountains/_mountains-fonts.scss:13]: let's remove commented code

File: docroot/themes/custom/twelve_carnation/twelve_carnation.theme [hunk 2]
```diff
@@ -43,5 +43,22 @@ function twelve_carnation_theme_suggestions_page_alter(array &$suggestions, arra
...
+   *
+   * Implements hook_library_info_alter().
+   */
+  function twelve_carnation_library_info_alter(&$libraries, $extension) {
+    if ($extension !== 'openy_carnation') {
+      return;
+    }
+
+    unset($libraries['global-styling']['css']['base']['//use.fontawesome.com/releases/v5.1.0/css/all.css']);
+    $libraries['global-styling']['css']['base']['//use.fontawesome.com/releases/v5.13.0/css/all.css'] = [
```
Comment (podarok) [inline docroot/themes/custom/twelve_carnation/twelve_carnation.theme:58]: why we can't update the theme itself?

**Comment by anpolimus** [comment]
@podarok we've fixed requested items.

File: docroot/modules/custom/twelve_app/twelve_app.module [hunk 3]
```diff
@@ -36,5 +36,50 @@ function twelve_app_theme() {
...
+  if (getenv('ENVIRONMENT') != 'development') {
+    return;
+  }
+
+  if ($extension != 'openy_system') {
+    return;
+  }
+
+  unset($libraries['vue']['js']['https://cdnjs.cloudflare.com/ajax/libs/vue/2.6.11/vue.min.js']);
+  $libraries['vue']['js']['https://cdnjs.cloudflare.com/ajax/libs/vue/2.6.11/vue.js'] = [
```
[the remark this answers is not in the document]
Comment (anpolimus) [reply docroot/modules/custom/twelve_app/twelve_app.module:66]: @podarok that is for our dev use only.
This hack works only at docksal instance and is disabled at production and demo

**Pull request closed without merging**